src/models/krate.rs

```diff
@@ -235,6 +240,21 @@ impl<'a> NewCrate<'a> {
 }
 
 impl Crate {
+    /// SQL filter with the `like` binary operator
+    /// ```sql
+    /// SELECT *
+    /// FROM crates
+    /// WHERE name like $1
+    /// ```
+    pub fn like_name(name: &str) -> LikeName<'_> {
+        canon_crate_name(crates::name).ilike(canon_crate_name(name))
```

Comment:
Just `like` is enough here, `canon_crate_name` already normalizes case

Reply:
I can change this back, the only reason I used `ilike` was from your previous comment.

Reply:
Sorry about that :(